app/handlers/handlers_resource.py

```diff
@@ -3,20 +3,24 @@
 import kopf
 
 from app.api import TwingateAPIClient
-from app.crds import K8sMetadata, ResourceSpec
+from app.crds import ResourceSpec
 from app.handlers.base import fail, success
 
 
 @kopf.on.create("twingateresource")
-def twingate_resource_create(body, meta, spec, memo, logger, patch, **kwargs):
-    logger.info("Got a create request: %s. Metadata: %s", spec, meta)
-    k8s_metadata = K8sMetadata(**meta)
+def twingate_resource_create(body, labels, spec, memo, logger, patch, **kwargs):
+    logger.info("Got a create request: %s. Labels: %s", spec, labels)
     resource = ResourceSpec(**spec)
     client = TwingateAPIClient(memo.twingate_settings, logger=logger)
+    labels = {
+        tag["name"]: tag["value"]
+        for tag in memo.twingate_settings.default_resource_tags
+    } | dict(labels)
```

Comment:
Why don't we move this conversation 
```
{
    tag["name"]: tag["value"]
    for tag in memo.twingate_settings.default_resource_tags
}
```
to `TwingateOperatorSettings` as [previously suggested](https://github.com/Twingate/kubernetes-operator/pull/613#discussion_r2045036542)? 🤔 It's repeated 3 times here.